pypoptools/pypoptesting/vbitcoind_node.py

```diff
@@ -175,6 +175,15 @@ def disconnect(self, node):
     def getnewaddress(self) -> str:
         return self.rpc.getnewaddress()
 
+    def getblockchaininfo(self) -> dict:
+        return self.rpc.getblockchaininfo()
+
+    def getpeerinfo(self) -> dict:
```

Comment:
```suggestion
    def getpeerinfo(self) -> List[PeerInfo]:
```

where PeerInfo contains:
- id: str
- banscore: int

Reply:
fixed